core/direct-rpc-server/src/rpc_responder.rs

```diff
@@ -142,18 +144,20 @@ pub mod tests {
 	use super::*;
 	use crate::{
 		builders::rpc_response_builder::RpcResponseBuilder,
-		mocks::{connection_mock::ConnectionMock, updates_sink::UpdatesSink},
+		mocks::response_channel_mock::ResponseChannelMock,
 		rpc_connection_registry::ConnectionRegistry,
 	};
-	use core::assert_matches::assert_matches;
+	use std::assert_matches::assert_matches;
```

Comment:
Using `core::` or `alloc::` requires an `extern crate core;` or `extern crate alloc;` statement in the crate. Whereas the `std;;` does not. So I prefer `std::` to reduce the amount of code needed.

Reply:
I agree about the `alloc` but `core` does not need any `extern crate` import, as far as I know?

Reply:
Maybe CLion does this wrong, but as soon as you import something from `core`, it will add the `extern crate` statement as well.

Reply:
Okay, good to know. Apparently the Rust plugin for CLion does not know that. As soon as you add a use with `core`, it will add the `extern crate` statement too.